src/Stratis.CirrusD/Program.cs

```diff
@@ -35,6 +35,8 @@ public static async Task MainAsync(string[] args)
         {
             try
             {
+                // set the console window title to identify this as a Cirrus full node (for clarity when running Strax and Cirrus on the same machine)
+                Console.Title = "Cirrus full node";
```

Comment:
Hi @0tim0, if you want this for the masternode's then should also be set in CirrusMinerD and CirrusPegD

Reply:
Also, please capitalize `full node`. Same with Strax below.

Reply:
@fassadlr Thanks for the feedback.  Updates made as requested